### ISSUE TITLE
Element missing end tag in <NuxtWelcome /> component after recent update.

### ISSUE THREAD

**Issue by youngcadam** [nuxt/nuxt#22383]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v16.20.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  npm@9.4.2
- Builder:          vite
- User Config:      devtools
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

Somewhat surprising it hasn't been noticed yet but this bug is occurring in the StackBlitz environment linked on the [Installation page](https://nuxt.com/docs/community/reporting-bugs#create-a-minimal-reproduction) in Nuxt's documentation  https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz 
![image](https://github.com/nuxt/nuxt/assets/17751995/5c2393ea-e251-4c02-b25e-fa4a55d4ce6a)
 and locally when I attempt to do the following 
```bash
npx nuxi init test
cd test
npm install
npm run dev -- -o
```

![image](https://github.com/nuxt/nuxt/assets/17751995/c72e4dd6-1c91-403d-a199-2b941d5c9425)

### Describe the bug

500 error caused by lack of end tag in `node_modules/@nuxt/ui-templates/dist/templates/welcome.vue`

![image](https://github.com/nuxt/nuxt/assets/17751995/1434dc26-3238-47db-9ac4-798402998726)
Seems to be related to https://github.com/nuxt/nuxt/commit/9741f267be1c04a69f3a25204c175f98d834565b and changes to `@nuxt/ui-templates` mixing with `vite-plugin-vue-inspector`
![image](https://github.com/nuxt/nuxt/assets/17751995/9acf1207-e939-4e63-b588-7782785722b9)
Safe to say I am not the only one experiencing this given it is occurring in one of nuxt's featured starter environments. 

### Additional context

Noticed this started happening when I build new nuxt applications in the past week or so and believe this coincides with these recent changes https://github.com/nuxt/nuxt/commit/9741f267be1c04a69f3a25204c175f98d834565b

Here is a screenshot of logs from browser that were too long to post:
![image](https://github.com/nuxt/nuxt/assets/17751995/bf8fcdd4-4a56-40f2-9e93-c3800fd3f94c)

### Logs

```shell-script
Logs from `npm run dev`output:
ERROR  Internal server error: Element is missing end tag.                                        3:50:08 PM
  Plugin: vite-plugin-vue-inspector
  File: C:/Users/ayoung.PRESCIENT/firefly/node_modules/@nuxt/ui-templates/dist/templates/welcome.vue
      at createCompilerError (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:18:17)
      at emitError (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:1487:5)
      at parseElement (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:1031:5)
      at parseChildren (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:844:18)
      at parseElement (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:1008:20)
      at parseChildren (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:844:18)
      at parseElement (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:1008:20)
      at parseChildren (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:844:18)
      at parseElement (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:1008:20)
      at parseChildren (C:\Users\ayoung.PRESCIENT\firefly\node_modules\@vue\compiler-core\dist\compiler-core.cjs.js:844:18) (x2)
```

**Comment by wondenge** [comment]
I was also bootstrapping a new app this evening and run to the same issue. I can confirm that we have the same errors. I can reproduce and provide more context if needed.

**Comment by LynxTR** [comment]
I am having this issue on cloudflare's pages deployment as well

**Comment by danielroe** [comment]
Thanks. This was triggered by a release of `@nuxt/ui-templates`. On it.

**Comment by ggoodwin** [comment]
For anyone looking for a workaround. Open this file.

node_modules>@nuxt>ui-templates>dist>templates>**welcome.vue**

Search the the following tag:
```
<nuxtwelcome>
```
and replace it with:
```
<nuxtwelcome />
```

Save and re-run the dev server script.

**Comment by danielroe** [comment]
The issue is already resolved upstream with release of v1.3.2.

**Comment by houssemFat** [comment]
Same error here using `bun`

`bun --version`

1.0.25

#### Log 
```
$ nuxt dev

Nuxt 3.10.0 with Nitro 2.8.1                                                                                                                                   9:41:58 PM
                                                                                                                                                               9:41:58 PM
  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose

  ➜ DevTools: press Shift + Option + D in the browser (v1.0.6)                                                                                                 9:41:59 PM


 ERROR  Pre-transform error: Element is missing end tag.                                                                                                       9:42:00 PM


 ERROR  Pre-transform error: Illegal tag name. Use '&lt;' to print '<'.                                                                                        9:42:00 PM

ℹ Vite client warmed up in 413ms                                                                                                                              9:42:00 PM
ℹ Vite server warmed up in 466ms                                                                                                                              9:42:00 PM
✔ Nitro built in 338 ms                                                                                                                                 nitro 9:42:00 PM


```

#### Borwser : 

<img width="1468" alt="image" src="https://github.com/nuxt/nuxt/assets/5431172/5b66dbbf-788d-47ac-9eef-7fd47e1448d8">
